docs/user-guides/server.md

```diff
@@ -487,117 +487,79 @@ r = c.encode(
 )
 ```
 
-[//]: # (## Deploy on JCloud)
+## Deploy on JCloud
 
-[//]: # ()
-[//]: # (You can deploy `CLIPTorchEncoder` on JCloud. )
+The `clip_server` can be smoothly deployed and hosted as a Flow on [JCloud](https://docs.jina.ai/fundamentals/jcloud/) to utilize the free computational and storage resources provided by Jina.
 
-[//]: # (A minimum YAML file `flow.yml` is as follows:)
+You need a YAML file to config the `clip_server` executor in the [Flow](https://docs.jina.ai/fundamentals/flow/) in order to deploy. 
+The executor is hosted on [Jina Hub](https://hub.jina.ai) and is sync with `clip_server` Python module. 
+We currently support [PyTorch-backed CLIP](https://hub.jina.ai/executor/gzpbl8jh).
```

Comment:
To serve the CLIP model with Pytorch runtime, you can use `CLIPTorchEncoder` in your flow.